marthamareal/ridemyway-api-django
# #163858323 Configure the project to use Postgres

#### What does this PR do?
- Adds Postgres as a database engine used in the project.
#### Description of Task to be completed?
- add postgres db settings
- add a .env file sample
- re-format the project files
#### How should this be manually tested?
- Fetch the branch to you local machine with ` git fetch origin ch-setup-postgres-db-163858323`
- Check out the branch `git checkout ch-setup-postgres-db-163858323`
- Edit the `.env.sample` file to `.env` and fill in the correct variables
- Run migrations with `python manage.py runserver`
- Check your database, you will find some tables created
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#163858323](https://www.pivotaltracker.com/story/show/163858323)
